src/lib/lite_base/blockchain_state.ml

```diff
@@ -1,10 +1,33 @@
 open Fold_lib
+open Module_version
 
-type t =
-  { staged_ledger_hash: Staged_ledger_hash.t
-  ; ledger_hash: Ledger_hash.t
-  ; timestamp: Block_time.t }
-[@@deriving bin_io, eq, sexp]
+module Stable = struct
+  module V1 = struct
+    module T = struct
+      type t =
+        { staged_ledger_hash: Staged_ledger_hash.t
```

Comment:
Do we have stable versions of `Staged_ledger_hash` and `Ledger_hash`; do we really need the `asserted`?

Reply:
Yes, because Staged_ledger_hash.t and Ledger_hash.t are derived from Pedersen_lib.Pedersen, which is not versioned as of right now.